dictionaries/git.json

```diff
@@ -17,13 +17,13 @@
 "TKPWEUT/KHEBG/O*UT": "git checkout",
 "TKPWEUT/KHEBG/O*UT/H-PBS/PW*": "git checkout -b",
 "TKPWEUT/KPHEUT/H-PBS/SR*": "git commit -v",
-"TKPWEUT/KPHEUT/H-PBS/SR*/TK-RB/APLD": "git commit -v --amend",
+"TKPWEUT/KPHEUT/H-PBS/SR*/H-PBZ/APLD": "git commit -v --amend",
 "TKPWEUT/PHERPBLG": "git merge",
 "TKPWEUT/ST*TS": "git status",
-"TKPWEUT/TKEUF/TK-RB/KAERBD": "git diff --cached",
```

Comment:
These two could be changed to `TK*RB` using the code dictionary, or `H-PBZ` as you've suggested, which matches Plover. Happy to run with that.

Reply:
Sounds good. So, I'll leave all the entries where `--` has been changed to `H-PBZ` for standardisation purposes.